### ISSUE TITLE
Normalize locale

### ISSUE THREAD

**Issue by XSlemX** [staccx/bento#28]
Parse locale and normalize for different masks and locales

**Comment by robinsandborg** [comment]
Could you elaborate? I may have some meanings about this 😂

**Comment by XSlemX** [comment]
input should be able to be "NO" and also "NB-no"

**Comment by robinsandborg** [comment]
Personally, I find language codes to be pretty confusing. In some instances you will find "NO" "NB-no" or "NB_no". Is there some kind of standard for this that is widely accepted? There seem to be several ISO standards about this, or that sometimes the country-code is used, while other times, the language code is used.

While it's very nice if the Inputs can accept several formats, I feel we should find one solution to be the "standard" for all this Bento – including sanity localization

**Comment by XSlemX** [comment]
Yes. Good idea. We can expose a `languages` from i18n. And they are all valid.

**Comment by XSlemX** [comment]
I think we will support only a subset of all the worlds locale in the start. Start with Scandinavia and G8?

**Comment by robinsandborg** [comment]
Sounds like a good place to start. Either that or making some way of adding locales as a part of the theme or something to that effect, so that each project can define it's needs and not have any sort of bloat from unused locales. Not sure if this is worth exploring though, as I'd guess each locale is pretty lightweight.
